src/python/py_generate_pipeline.cpp

```diff
@@ -419,6 +420,23 @@ PYBIND11_MODULE(py_generate_pipeline, m) {
             Add {"scheduler_config": ov_genai.SchedulerConfig} to config properties to create continuous batching pipeline.
         )")
 
+        .def(py::init([](
+            const std::string& model_path, 
+            const std::string& device,
+            const py::kwargs& kwargs
+        ) {
+            ScopedVar env_manager(utils::ov_tokenizers_module_path());
+            return std::make_unique<LLMPipeline>(model_path, device, utils::kwargs_to_any_map(kwargs));
+        }),
+        py::arg("model_path"), "folder with openvino_model.xml and openvino_tokenizer[detokenizer].xml files", 
+        py::arg("device") = "CPU", "device on which inference will be done",
+        R"(
+            LLMPipeline class constructor.
+            model_path (str): Path to the model file.
+            device (str): Device to run the model on (e.g., CPU, GPU). Default is 'CPU'.
+            Add {"scheduler_config": ov_genai.SchedulerConfig} to config properties to create continuous batching pipeline.
+        )")
```

Comment:
@popovaan here several ctors for LLMPipeline has been added.
Please, consider merging them according to your suggestion with default args for `config` during your work on alignment of python ctors for GenAI pipeline